client/src/app/gui.rs

```diff
@@ -128,14 +128,15 @@ impl Gui {
                         .add(button)
                         .on_hover_text("Discard previously captured packets");
                     if resp.clicked() {
-                        self.packets.borrow_mut().clear();
+                        self.streams.borrow_mut().clear();
```

Comment:
No need to clear self.packets as well?

Reply:
If you looked at implementation of `Streams::clear`, you'd know that it clears packets as well.